test/viz/index.spec.ts

```diff
@@ -10,6 +10,10 @@ console.log = jest.fn();
 const fiveMinutes = 5 * 60 * 1000;
 jest.setTimeout(fiveMinutes);
 
+const hasFile = async (...paths: string[]): Promise<boolean> => {
```

Comment:
I think this functionality is already present in the `src/files.ts` file.

Reply:
hmm okay. This was just moving around code already in the viz test.